src/ua.c

```diff
@@ -1317,11 +1317,8 @@ int ua_connect_dir(struct ua *ua, struct call **callp,
 		goto out;
 
 	if (adir != SDP_SENDRECV || vdir != SDP_SENDRECV) {
-		err = call_set_media_direction(call, adir, vdir);
-		if (err) {
-			mem_deref(call);
-			goto out;
-		}
+		(void)call_set_media_estdir(call, adir, vdir);
+		call_set_media_direction(call, adir, vdir);
```

Comment:
We removed returned error because we don't care about the media direction if `call==NULL`, generally.
In my option we should also remove the error code for `call_set_media_estdir()`.
This PR already breaks the ABI because it replaces `call_sent_answer()` by `call_sdp_neg_state()` and `call_sdp_change_allowed()`.
Any other opinions?